test_plugin/src/main/java/io/github/madethoughts/mayflower/testplugin/TestPlugin.java

```diff
@@ -19,11 +19,12 @@
 import io.github.madethoughts.mayflower.lifecycle.event.EnableEvent;
 import io.github.madethoughts.mayflower.listener.McListener;
 import io.github.madethoughts.mayflower.plugin.MayflowerPlugin;
-import io.github.madethoughts.mayflower.plugin.PaperPlugin;
+import io.github.madethoughts.mayflower.plugin.McPlugin;
 import io.micronaut.runtime.event.annotation.EventListener;
 import org.bukkit.event.player.PlayerJoinEvent;
 
-@PaperPlugin
+@SuppressWarnings("ALL")
+@McPlugin(name = "TestPlugin", version = "0.1", apiVersion = McPlugin.ApiVersion.V1_19, authors = "goldmensch")
```

Comment:
The smoke test depends on that name
```suggestion
@McPlugin(name = "MayflowerTestPlugin", version = "0.1", apiVersion = McPlugin.ApiVersion.V1_19, authors = "goldmensch")
```